user_interactions/user_functions.py

```diff
@@ -48,5 +49,30 @@ def deleteStudentByID():
     
 
 
-def studentAge(student :Student):
-    pass
\ No newline at end of file
+def studentAge():
+    # printing msgs and taking input id from the user
+    print("*"*20,'calculate age of a student ',"*"*20)
+    print("calculating age of a students by ID..")
+    id = input("Enter the ID of a student you want to calculate age of ==>")
+    # check for the student having the id,,
+    students = Student.view_students()
+    student_id = None
+    for student in students:
```

Comment:
You can add filter function here. Please make changes.

Reply:
I will make changes next time...